app/v2/incidents.py

```diff
@@ -67,12 +67,12 @@ def post(self):
             return error(400, state[1])
 
 
-class RedFlag(RedFlags):
+class Incident(Incidents):
```

Comment:
Don't inherit like this, since it is a new resource just
```suggestion
class Incident(Resource):
```